.gitignore

```diff
@@ -5,3 +5,7 @@ java_pid*.hprof
 local.properties
 out/
 venv/
+
+# Samples folders -> content is generated by running ./gradlew generateSwagger
+samples/kotlin-android-with-moshi-kotlin-codegen/src/main/java/
+samples/kotlin-coroutines/src/main/java/
```

Comment:
We do have local `.gitignore` files for other samples so we should probably do the same for those two examples.